info.json

```diff
@@ -21,17 +21,17 @@
         }
     ],
     "prerequisite": "",
-    "certified": "false",
```

Comment:
why this is removed?

Reply:
by giving certified value as false it is showing Certified as Yes
![Screenshot 2023-03-01 at 4 46 27 PM](https://user-images.githubusercontent.com/105705710/222124360-7354f0bd-9e59-4964-9f68-25bc46e111fa.png)